src/service/resources-impl.js

```diff
@@ -254,6 +258,22 @@ export class Resources {
       this.schedulePass();
       this.monitorInput_();
     });
+
+    // Safari 10 and under incorrectly estimates font spacing for `@font-face`
```

Comment:
"Documented" value is 3000ms. In all my tests I got 3050ms. So, I added a bit on top to 3100ms. Kind of really "rule of thumb" thing, but the idea is to get a "fair effort" to remeasure.